Tests/fixtures/MutationExamples/SideEffect/sampleWithImplicitReturn.swift

```diff
@@ -0,0 +1,17 @@
+func mutateMe() {
+    bar()
+}
+
+func mutateMe2() -> Void {
+    bar()
+}
+
+func bar() { }
+
+func ignoreMe() -> Int {
+    returnInt()
```

Comment:
it might be a good idea to include some negative examples as well, like `return returnInt()`, just to make sure it's working both ways